tests/python/unittest/test_tir_op_types.py

```diff
@@ -203,11 +229,15 @@ def test_tir_op_TVMBackendFreeWorkspace():
     test_tir_op_type_annotation()
     test_tir_op_tvm_access_ptr()
     test_tir_op_tvm_throw_last_error()
-    test_tir_op_tvm_load_matrix_sync(),
-    test_tir_op_tvm_store_matrix_sync(),
-    test_tir_op_tvm_mma_sync(),
-    test_tir_op_tvm_bmma_sync(),
-    test_tir_op_tvm_fill_fragment(),
+    test_tir_op_tvm_load_matrix_sync()
```

Comment:
Would be great if we could use `tvm.testing.main()`

Reply:
updated